src/khoj/processor/conversation/prompts.py

```diff
@@ -294,6 +299,40 @@
 """.strip()
 )
 
+pick_relevant_mode = PromptTemplate.from_template(
```

Comment:
nit: We should make the name more specific to its purpose

```suggestion
pick_relevant_output_mode = PromptTemplate.from_template(
```